extensions/Oisthebestletter/sb3ai.js

```
@@ -0,0 +1,481 @@
+            text: "generate [PROMPT]",
+            arguments: {
+              PROMPT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "Who was Alan Turing?",
+              },
+            },
+          },
+          {
+            opcode: "unmask",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "unmask [MASKED_TEXT]",
```

ESLint

[error] Block text should usually be translated

(extension/should-translate)